.github/workflows/shadow-repo.yml

```diff
@@ -0,0 +1,72 @@
+name: Sync with shadow repo
+
+on:
+  push:
+  create:
+  delete:
+  pull_request:
+    types: [opened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
@jerivas It seems like this is causing it to be canceled when a push happens on a different branch?